Source/Core/VulkanConstants.js

```diff
@@ -0,0 +1,287 @@
+/**
+ * Enum containing Vulkan Constant values by name.
+ * for use when needing VK Formats
+ *
+ * These match the constants from the [Vulkan 1.2]{@link https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#formats-definition}
+ * specifications.
+ *
+ * @exports VulkanConstants
```

Comment:
Since https://github.com/CesiumGS/cesium/pull/8878 this should now be `@enum {Number}`

Reply:
Updated